code/headers/dof4_diy_c.hpp

```diff
@@ -0,0 +1,23 @@
+//
+// Created by Slimmerik on 13/04/2019.
+//
+#pragma once
+
+#include <hwlib.hpp>
+#include <robot_arm_interface_c.hpp>
+#include <vector3_c.hpp>
+
+
+
+namespace r2d2::robot_arm{
+    class dof4_diy : public robot_arm_interface{
+    public:
+
+        void move_head_to_coordinate(const vector3_c< uint16_t > &v);
+
+        void rotate_head(const uint16_t &r);
```

Comment:
Please add override if method `overrides` function from interface. Also, I believe that `uint16_t` should be `int16_t` since a negative value can be passed.